File: CppCompiler/for5.cpp
#include<iostream>
using namespace std;

int main()
{
 int i,n,sum=0;
cout<<"enter no";
cin>>n;
for(i=1;i<=10;i++)
{
cout<<n<<"*"<<i<<"="<<n*i<<endl;
sum=sum+n*i;
}
cout<<sum;
return 0;
}

File: JavaNIDE/volume/app/src/main/java/sample/For5.java
package sample;
import java.util.*;

class For5 
{
  static int  i,j,fact=1,n,z;
  static float sum=0,sum1;
  public static void main(String[] ar)
  {
    Scanner m=new Scanner(System.in);
    System.out.print("enter the last term");
    n=m.nextInt();
    for(i=1;i<=n;i++)
    {
     System.out.print(i+"-"+i+"!"+"/");
     for(j=1;j<=i;j++)
     {
      fact=fact*j;
     }
    z=(i*i)*(i+1)*(i+1);
    sum1=(float)fact/z;
    sum=sum+(i-sum1);
    System.out.print(i+"*"+i+"+"+(i+1)+"*"+(i+1));
     System.out.print(" + ");
     fact=1;
    }
    
    System.out.print("sum of seriese is"+sum);
  }
}
